.gitlab-ci.yml

```diff
@@ -1051,6 +1052,45 @@ test_full_integration:
     reports:
       junit: results_full_integration.xml
 
+# This job is to allow mender master and release branches to publish the related Docker images
```

Comment:
I would write "...publish the related Docker **client** images". I was a bit confused before I looked at the pipeline name.

Reply:
I rewrote it and also renamed `PUBLISH_CLIENT_DOCKER_IMAGES` to `PUBLISH_DOCKER_CLIENT_IMAGES`